.github/workflows/binaries-and-deb-release.yml

```diff
@@ -0,0 +1,618 @@
+            }
+          }
+          EOF
+
+      - name: Generate SPDX SBOM for Debian package
+        env:
+          PACKAGE_ARCH: ${{ matrix.arch }}
+        run: |
+          python3 <<'PY'
+          import hashlib
+          import os
+          from pathlib import Path
+
+          package_path = Path(f"zallet_{os.environ['RELEASE_VERSION']}_{os.environ['PACKAGE_ARCH']}.deb")
+          if not package_path.exists():
+              raise SystemExit(f"Missing package at {package_path}")
+          checksum = hashlib.sha256(package_path.read_bytes()).hexdigest()
+          output = package_path.parent / f"{package_path.name}.sbom.spdx"
+          doc = f"""SPDXVersion: SPDX-2.3
+          DataLicense: CC0-1.0
+          SPDXID: SPDXRef-DOCUMENT
+          DocumentName: {package_path.name}
+          DocumentNamespace: https://github.com/{os.environ['GITHUB_REPOSITORY']}/sbom/{package_path.name}-{checksum}
+          Creator: Tool: zallet-ci-spdx-1.0
+          Created: 1970-01-01T00:00:00Z
+          PackageName: {package_path.name}
+          PackageSPDXID: SPDXRef-Package-{checksum[:12]}
+          PackageDownloadLocation: NOASSERTION
+          FilesAnalyzed: false
+          PackageChecksum: SHA256: {checksum}
+          """
+          output.write_text(doc, encoding="utf-8")
+          PY
+
+      - name: Generate build provenance attestation for .deb
+        id: attest
+        uses: actions/attest-build-provenance@977bb373ede98d70efdf65b84cb5f73e068dcc2a # v3.0.0
+        with:
+          subject-path: zallet_${{ env.RELEASE_VERSION }}_${{ matrix.arch }}.deb
+
+      - name: Save .deb attestation bundle
+        env:
+          BUNDLE_PATH: ${{ steps.attest.outputs['bundle-path'] }}
+        run: |
+          cp -- "$BUNDLE_PATH" "zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb.intoto.jsonl"
+
+      - name: Collect release deliverables
+        run: |
+          set -euo pipefail
+          DEST="release/${{ matrix.platform_key }}"
+          mkdir -p "$DEST"
+          cp standalone/* "$DEST"/
+          cp zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb "$DEST/"
+          cp zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb.asc "$DEST/"
+          cp zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb.sbom.spdx "$DEST/"
+          cp zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb.intoto.jsonl "$DEST/"
+          cp zallet_${RELEASE_VERSION}_${{ matrix.arch }}.deb.provenance.json "$DEST/"
+
+      - name: Upload artifacts for publish job
+        uses: actions/upload-artifact@330a01c490aca151604b8cf639adc76d48f6c5d4 # v5.0.0
+        with:
+          name: release-${{ matrix.platform_key }}
+          path: release/${{ matrix.platform_key }}/*
+
+  publish:
+    name: Create/Update GitHub Release (aggregate)
+    needs: binaries_deb_release
+    if: ${{ github.ref_type == 'tag' }}
```

Comment:
Does this work now that this job is triggered by `workflow_call`? (In `deb-release` it was triggered by the tag directly, but that is now handled by `release.yml`.)